frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-inventory-card.tsx

```diff
@@ -56,6 +66,12 @@ export const VMInventoryCard: React.FC<VMInventoryCardProps> = () => {
           count={diskCount}
           TitleComponent={DisksTitle}
         />
+        <InventoryItem
+          isLoading={isLoading}
+          title="CD-ROM"
+          count={cdromCount}
+          TitleComponent={CDROMTitle}
```

Comment:
@matthewcarleton @Ranelim Hi, 

This PR adds a "CD ROMS" inventory item to the VM invertory card in the dashboard.

The current behaviour is to link to the details page (since we do not have a dedicated "cdroms" tab), this is suboptimal because "disks" and "nics" link to dedicated tabs.

The question is - what can we do to fix that ?
Screenshot:
![Peek 2020-02-19 13-17](https://user-images.githubusercontent.com/2181522/74833752-c35f0800-5322-11ea-9099-2ba2c2252890.gif)